MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -2043,10 +2043,11 @@ InstrumentWindow* MantidUI::getInstrumentView(const QString & wsName, int tab)
 
   //Need a new window
   const QString windowName(QString("InstrumentWindow:") + wsName);
-  InstrumentWindow *insWin = new InstrumentWindow(wsName,QString("Instrument"),appWindow(),windowName);
+  InstrumentWindow *insWin = new InstrumentWindow(wsName, QString("Instrument"), appWindow(), windowName);
+
   try
   {
-    insWin->init();
+    insWin->getInstrumentWidget()->init();
```

Comment:
Worth investigating if the `init` can't somehow be wrapped in the constructor. If this is not possible, I would do the construction via a `FactoryMethod` to ensure that the `InstrumentWidget` you get back is properly setup.